tests/testAstrometrySourceSelector.py

```diff
@@ -17,15 +18,15 @@
 #
 # You should have received a copy of the LSST License Statement and
 # the GNU General Public License along with this program.  If not,
-# see <https://www.lsstcorp.org/LegalNotices/>.
+# see <http://www.lsstcorp.org/LegalNotices/>.
 #
 from __future__ import division, absolute_import, print_function
 
 import unittest
 import numpy as np
 
 import lsst.utils
-from lsst.meas.base.tests import TestDataset
+from lsst.meas.base.tests import TestDataset as Dataset
```

Comment:
Renaming symbols on import is confusing. I understand why you can't import the bare name using its original name, but I'd suggest using the package name as a prefix instead.